Ports/ObjFW/patches/0001-Disable-pledge.patch

```diff
@@ -0,0 +1,19 @@
+Disable pledge() support in ObjFW
+
+When ObjFW detects pledge(), it expects it to be exactly OpenBSD-
+compatible, which ours is not. This then causes ObjFW to hard-abort
+upon trying to pledge().
+
+--- a/configure.ac
++++ b/configure.ac
+@@ -1994,10 +1994,6 @@
+ AC_CHECK_HEADERS_ONCE([complex.h sys/ioctl.h sys/ttycom.h])
+ AC_CHECK_FUNCS(ioctl isatty)
+ 
+-AC_CHECK_FUNC(pledge, [
+-	AC_DEFINE(OF_HAVE_PLEDGE, 1, [Whether we have pledge()])
+-])
+-
```

Comment:
I would prefer setting `ac_cv_func_pledge=no` in the package script, patching the `pledge` calls themselves, or at the very least removing the check in `configure` instead of `configure.ac`.

Reply:
I would advise against patching configure: If the next release is generated with a newer autoconf version, the patch is basically guaranteed to fail.

Reply:
While autoconf output looks quite funky, small isolated chunks are usually quite stable. If we wanted to, we could take our chances. But yeah, the first two are obviously preferable.

Reply:
Went with `ac_cv_func_pledge=no`